tests/Api/ResourceIterationEventsTest.php

```diff
@@ -0,0 +1,131 @@
+<?php
```

Comment:
Please add the copyright header to all files.

```php
<?php

declare(strict_types=1);

/*
 * This file is part of the Gitlab API library.
 *
 * (c) Matt Humphrey <matth@windsor-telecom.co.uk>
 * (c) Graham Campbell <hello@gjcampbell.co.uk>
 *
 * For the full copyright and license information, please view the LICENSE
 * file that was distributed with this source code.
 */

namespace Gitlab
```

Reply:
Added in 026697f64a51f75bb4ff1dc66e87b1ee06b366ce